conda-recipe/meta.yaml

```diff
@@ -1,38 +1,44 @@
-{% set data = load_setup_py_data() %}
+{% set package_name = "transfocate" %}
+{% set import_name = "transfocate" %}
+{% set version = load_file_regex(load_file=os.path.join(import_name, "_version.py"), regex_pattern=".*version = '(\S+)'").group(1) %}
 
 package:
-  name: transfocate
-  version : {{ data.get('version') }}
+  name: {{ package_name }}
+  version: {{ version }}
 
 source:
   path: ..
 
 build:
   number: 0
   noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
 
 requirements:
   build:
-    - python >=3.6
-    - setuptools
-
+  - python >=3.9
+  - setuptools_scm
+  - pip
   run:
-    - python >=3.6
-    - bluesky
-    - databroker
-    - numpy
-    - openpyxl
-    - ophyd >=1.5.0
-    - pandas
-    - pcdsdevices >=4.0.0
-    - prettytable
-    - reportlab
+  - python >=3.9
+  - bluesky
+  - databroker
+  - matplotlib-base
```

Comment:
Is there a story behind this dependency addition as well? Were we importing matplotlib and not declaring it here previously?

Reply:
It's an "optional" dependency, used in multiple modules which might not be imported normally.

I think it's appropriate to list everything in the recipe here, but I could probably be convinced otherwise

Reply:
I think it's appropriate too, note that some of the other dependencies also depend on matplotlib so it's largely an advisory/clarity dependency.